server/util/filters.js

```diff
@@ -112,6 +112,10 @@ exports.ftdate = function(date) {
   });
 };
 
+exports.toOneDP = function (number, places) {
+  return (Math.round(number * 10) / 10) + '';
```

Comment:
* Places is not used - this would be more useful if took a places 
* and style: why do `+ ''` when `toString()` is much clearer?